repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -262,20 +258,20 @@ object KeystoneRequestHandler {
     }
   }
 
-  trait IdentityException
+  abstract class IdentityException(message: String, cause: Throwable) extends Exception(message, cause)
```

Comment:
man this was weird before

Reply:
Right? I don't remember why this was a trait... I can kind of see that for classifying exceptions but still allowing them to call different constructors on `Exception`, but we always call the same constructor here, and make the base exception a class seems more natural.